src/hooks/useCollectible.ts

```diff
@@ -59,17 +53,32 @@ function useRevalidateInBackground({
   isExternal: boolean;
   enabled: boolean;
 }) {
-  const dispatch = useDispatch();
+  const { accountAddress } = useAccountSettings();
   useEffect(() => {
+    const updateNFT = async () => {
+      const simplehashNFT = await fetchSimpleHashNFT(contractAddress!, tokenId);
+      const updatedNFT =
+        simplehashNFT && simpleHashNFTToUniqueAsset(simplehashNFT);
+
+      if (updatedNFT) {
```

Comment:
BEFORE:
refetch specific nft from opensea and replace that nft in redux storage w/ the new version

AFTER:
refetch specific nft from simplehash and replace that nft in the react query cache